client/src/client/frame-runner.js

```diff
@@ -1,8 +1,10 @@
 import '@babel/polyfill';
 import jQuery from 'jquery';
 import curriculumHelpers from '../utils/curriculum-helpers';
+import cssTree from 'css-tree';
 
 window.$ = jQuery;
+window.cssTree = cssTree;
```

Comment:
If you put this in the same place as curriculumHelpers it'll be available during testing, but you won't be able to use it in a challenge.  You can confirm this by entering `<script> console.log(window.$) </script>` into a challenge and comparing that with `<script> console.log(window.__helpers) </script>`

Reply:
It's 989kb unpacked (not sure if that's big or not).

I could load it in to helpers, sure - my thought process was that being able to access `cssTree` instead of `__helpers.cssTree` might be a smoother DX, but if there's concerns about loading it into the window I can change that. 🙂